Dockerfile

```diff
@@ -18,14 +18,14 @@ RUN sh /uv-installer.sh && rm /uv-installer.sh
 # Ensure the installed binary is on the `PATH`
 ENV PATH="/root/.local/bin/:$PATH"
 
-# Copy the project into the image
-ADD . /app
+## Copy the project into the image
+ADD ./pyproject.toml ./uv.lock ./README.md ./ee-zeno-service-account.json /app/
```

Comment:
If `ee-zeno-service-account.json` is a secret file, we should not copy it into the docker file -- it should be mounted as a volume instead.

Reply:
I think we can get rid of that with the new agents, we are no longer using GEE.